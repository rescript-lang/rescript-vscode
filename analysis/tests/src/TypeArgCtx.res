type someTyp = {test: bool}
let catchResult = Support.CatchResult.Ok({
  value: {
    test: true,
  },
})

// switch catchResult { | Ok({value: }) => ()
//                                  ^com
