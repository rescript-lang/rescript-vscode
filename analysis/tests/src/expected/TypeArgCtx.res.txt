Complete src/TypeArgCtx.res 7:36
posCursor:[7:36] posNoWhite:[7:35] Found pattern:[7:26->7:39]
Ppat_construct Ok:[7:26->7:28]
posCursor:[7:36] posNoWhite:[7:35] Found pattern:[7:29->7:38]
Completable: Cpattern Value[catchResult]->variantPayload::Ok($0), recordField(value)
Package opens Pervasives.JsxModules.place holder
Resolved opens 1 pervasives
ContextPath Value[catchResult]
Path catchResult
[{
    "label": "{}",
    "kind": 22,
    "tags": [],
    "detail": "someTyp",
    "documentation": {"kind": "markdown", "value": "```rescript\ntype someTyp = {test: bool}\n```"},
    "sortText": "A",
    "insertText": "{$0}",
    "insertTextFormat": 2
  }]

